src/lib/pages/frontsite/OpeningHours/OpeningHours.js

```diff
@@ -0,0 +1,83 @@
+import React, { Component } from 'react';
+import Overridable from 'react-overridable';
+import { Container, Grid, Header, Icon } from 'semantic-ui-react';
+import PropTypes from 'prop-types';
+import { Error } from '@components/Error';
+import { Loader } from '@components/Loader';
+import LocationOpeningHours from './LocationOpeningHours';
+import { sessionManager } from '@authentication/services/SessionManager';
+import _get from 'lodash/get';
+import { AuthenticationGuard } from '@authentication/components/AuthenticationGuard';
+import { Link } from 'react-router-dom';
+import { BackOfficeRoutes } from '@routes/backoffice/backofficeUrls';
+
+class OpeningHours extends Component {
+  constructor(props) {
+    super(props);
+    this.fetchAllLocations = props.fetchAllLocations;
+  }
+
+  componentDidMount() {
+    this.fetchAllLocations();
+  }
+
+  renderItems = () => {
+    const {
+      data: { hits },
+    } = this.props;
+    const locationPid = _get(sessionManager, 'user.locationPid');
+    return hits
+      .slice()
+      .sort((a, b) =>
+        a.pid === locationPid
+          ? -1
+          : b.pid === locationPid
+          ? 1
+          : ('' + a.pid).localeCompare(b.pid)
+      )
+      .map(location => (
+        <Grid.Row key={location.pid}>
+          <Grid.Column stretched>
+            <Header as="h3">
+              {location.metadata.name}{' '}
+              <AuthenticationGuard
```

Comment:
not a blocker: this might be a bit slow, it will check per each location the roles. I would just maybe save the roles in the state and check them once?